lib/tasks/eslint.rake

```diff
@@ -7,6 +7,7 @@ namespace :eslint do
   desc %{Run ESLint against the specified JavaScript file and report warnings (default is 'application')}
   task :run, [:filename] => :environment do |_, args|
     warnings = ESLintRails::Runner.new(args[:filename]).run
+    puts
```

Comment:
This one's actually here intentionally. There are some print lines inside of the the runner that output a sort of progressbar, with `.`, `H`, or `L` depending on the status of the file. This adds a newline after that. Maybe there's a better spot for it.